data/core/units/monsters/Skeletal_Dragon.cfg

```diff
@@ -5,21 +5,26 @@
     #not 'race=monster', because we need the not_living attribute
     race=undead
     image="units/monsters/skeletal-dragon/skeletal-dragon.png"
-    hitpoints=86
-    movement_type=undeadfly
-    movement=5
-    experience=200
+    hitpoints=75
+    movement_type=undeadfoot
+    movement=4
+    experience=220
     level=4
     alignment=chaotic
     advances_to=null
     {AMLA_DEFAULT}
     cost=100
+    [abilities]
+    #Other undead beings have the submerge ability, maybe change to not have this
+	  {ABILITY_SUBMERGE}
```

Comment:
I think it was argued extensively why we shouldn't have this for this unit

Reply:
Can you reiterate the arguments, please? I don't fully remember.

Reply:
Summary: Unit is "Too big" (dragon) so submerging on a bridge over deep water or a small patch of deep water makes little sense.

Reply:
Hmm… that _is_ a reasonably convincing argument. Okay, sure.

Reply:
You should also consider that this is a boss monster in campaigns. It would be confusing when such a large monster is invisible when the player encounters it. So please no submerge for Skeletal Dragon.

Reply:
But, on the other hand, it has undeadfoot and that means 10 % defense in deep water. In that case, submerge would be absolutely fine ...